Recurly/Resource.cs

```diff
@@ -5,6 +5,7 @@ namespace Recurly
     [ExcludeFromCodeCoverage]
     public class Resource
     {
+        public Response Response { get; set; }
```

Comment:
Couple things to note:

1. Since we are exposing this, we need to document it. Let's add a section to the README similar to the docs we have for this feature in other languages
2. I'm okay with the `Response` property but this means that we can no longer support the key name `response` in json and we'd need to add this to the recurly-api-spec checker. My compromise for the other languages, while not idomatic, was to use a getter naming scheme `getResponse()`, `get_response`, `GetResponse()` etc.

Reply:
Updated README and adjusted the property